Add function to alter sampling bounds on parameters to specified value. (#490)
open-AIMS/ADRIA.jl: docs/src/usage/scenario_generation.md

    coral_cover_tol=1.0
)
```
The ranges for sampling can also be constrained if running scenarios for a particular 
range of a parameter is of interest. For example, if one wanted to investigate only 
scenarios with high fogging and seeding, the following example could be used:

```julia
dom = ADRIA.load_domain()

# Adjust seeding bounds. Note only lower and upper bounds needed because it is a 
# uniform distribution.
ADRIA.set_factor_bounds!(dom, :N_seed_TA, (500000.0, 1000000.0 + 1.0))
ADRIA.set_factor_bounds!(dom, :N_seed_CA, (500000.0, 1000000.0 + 1.0))

> ADRIA.set_factor_bounds!(dom, :N_seed_SA, (500000.0, 1000000.0 + 1.0))

Reviewer: Any way of setting bounds for a group of factors like in the `fix_factor` example above?

Author: There is a function for this but I just added an example to the markdown

    coral_cover_tol=1.0
)
```

> The ranges for sampling can also be constrained if running scenarios for a particular 
> range of a parameter is of interest. For example, if one wanted to investigate only 
> scenarios with high fogging and seeding, the following example could be used:

Reviewer: ```suggestion
Sampling over a constrained range is also possible. For example, if one wanted to investigate only 
scenarios with high fogging and seeding, the following could be used:
```

```julia
dom = ADRIA.load_domain()

> # Adjust seeding bounds. Note only lower and upper bounds needed because it is a 

Reviewer: ```suggestion
# Adjust seeding bounds. Note only lower and upper bounds are needed because the factors in question have a 
```

# uniform distribution.
ADRIA.set_factor_bounds!(dom, :N_seed_TA, (500000.0, 1000000.0 + 1.0))
ADRIA.set_factor_bounds!(dom, :N_seed_CA, (500000.0, 1000000.0 + 1.0))
ADRIA.set_factor_bounds!(dom, :N_seed_SA, (500000.0, 1000000.0 + 1.0))

# Adjust fogging bounds. Note lower, upper and mode parameters are needed because it 

> # is a triangualr distribution.

Reviewer: ```suggestion
# is a triangular distribution.
```

ADRIA.set_factor_bounds!(dom, :fogging, (0.2, 0.3, 0.1))

> # Adjust multiple parameter in one function evaluation

Reviewer: ```suggestion
# Adjust multiple factors in one function evaluation
```

> # Adjust multiple factors in one function evaluation

Reviewer: I know it is one "function evaluation" but it's not the usual language in this context
